src/vs/workbench/contrib/issue/browser/issueQuickAccess.ts

```diff
@@ -0,0 +1,156 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { PickerQuickAccessProvider, IPickerQuickAccessItem, FastAndSlowPicks, Picks, TriggerAction } from 'vs/platform/quickinput/browser/pickerQuickAccess';
+import { IContextKeyService } from 'vs/platform/contextkey/common/contextkey';
+import { IMenuService, MenuId, MenuItemAction, SubmenuItemAction } from 'vs/platform/actions/common/actions';
+import { matchesFuzzy } from 'vs/base/common/filters';
+import { IQuickPickSeparator } from 'vs/platform/quickinput/common/quickInput';
+import { localize } from 'vs/nls';
+import { ICommandService } from 'vs/platform/commands/common/commands';
+import { IExtensionService } from 'vs/workbench/services/extensions/common/extensions';
+import { IConfigurationService } from 'vs/platform/configuration/common/configuration';
+import { IRelaxedExtensionDescription } from 'vs/platform/extensions/common/extensions';
+import { ThemeIcon } from 'vs/base/common/themables';
+import { Codicon } from 'vs/base/common/codicons';
+import { IssueSource } from 'vs/platform/issue/common/issue';
+import { IProductService } from 'vs/platform/product/common/productService';
+
+export class IssueQuickAccess extends PickerQuickAccessProvider<IPickerQuickAccessItem> {
+
+	static PREFIX = 'issue ';
+
+	constructor(
+		@IMenuService private readonly menuService: IMenuService,
+		@IContextKeyService private readonly contextKeyService: IContextKeyService,
+		@ICommandService private readonly commandService: ICommandService,
+		@IExtensionService private readonly extensionService: IExtensionService,
+		@IConfigurationService private readonly configurationService: IConfigurationService,
+		@IProductService private readonly productService: IProductService
+	) {
+		super(IssueQuickAccess.PREFIX, { canAcceptInBackground: true });
+	}
+
+	protected override _getPicks(filter: string): Picks<IPickerQuickAccessItem> | FastAndSlowPicks<IPickerQuickAccessItem> | Promise<Picks<IPickerQuickAccessItem> | FastAndSlowPicks<IPickerQuickAccessItem>> | null {
+		if (this.configurationService.getValue<string>('extensions.experimental.quickAcessExtensions')) {
+			const issuePicks: Array<IPickerQuickAccessItem | IQuickPickSeparator> = [];
+			const extensionIdList: Array<string> = [];
+
+			// add regular open issue reporter button
+			const createTerminalLabel1 = this.productService.quality === 'stable' ? localize("workbench.action.openIssueReporter", "Visual Studio Code") : localize("workbench.action.openIssueReporterInsiders", "Visual Studio Code: Insiders");
+			issuePicks.push({
+				label: `$(plus) ${createTerminalLabel1}`,
```

Comment:
I don't think we need the pluses tbh. They'll be separated from the extension ones. It'll be clear